packages/bazel-bot/docker-image/generate-googleapis-gen.sh

```diff
@@ -141,12 +141,15 @@ for (( idx=${#ungenerated_shas[@]}-1 ; idx>=0 ; idx-- )) ; do
         git -C "$GOOGLEAPIS_GEN" tag "googleapis-$sha"
         git -C "$GOOGLEAPIS_GEN" push origin "googleapis-$sha"
     else
+        # Determine the current branch so we can explicitly push to it
+        declare -r googleapis_gen_branch=$(git -C "$GOOGLEAPIS_GEN" branch --show-current)
```

Comment:
where does the actual clone take place, I'm guessing this will need to be updated too?

Reply:
Aha - just found that. It happens in docker-main.sh. So we'll need to parameterize that as well somehow. I'm happy to try to work that into this PR, or do it separately - let me know.

Reply:
I'd be happy with it in this same PR personally.

Reply:
Righto. I'll get back to this next week.

Reply:
Added a second commit to allow the clone to be controlled via environment variables. I'd *expect* that to be all that's required. PTAL.